nbgrader/exchange/ngshare/list.py

```diff
@@ -221,11 +191,9 @@ def _unrelease_assignment(self, course_id, assignment_id):
         """
         Unrelease a released assignment.
         """
-        url = '{}{}/assignment/{}/{}'.format(self.ngshare_url, self.prefix,
-                                             course_id, assignment_id)
+        url = '/assignment/{}/{}'.format(course_id, assignment_id)
 
-        response = requests.delete(url)
-        self.check_response(response)
+        self.ngshare_api_delete(url)
```

Comment:
Capture return value here and change try-except block to if-statement in lines 474-481.

Reply:
Fixed.